### PR TITLE
vim version

### PR DESCRIPTION
**Pull request by amirbilu**
- switch to vim version
- switch to vim version

### CODE REVIEW

**Comment by amirbilu** [comment]
@aarondill can you please approve? this is following https://github.com/codota/tabnine-nvim/pull/79

File: lua/tabnine/binary.lua [hunk 1]
```diff
@@ -41,11 +41,10 @@ local function binary_path()
 	end, fn.glob(binaries_path .. "/*", true, true))
 
 	paths = vim.tbl_map(function(path)
-		return semver(path)
+		return semver.parse(path)
 	end, paths)
 
 	table.sort(paths)
```
Comment (aarondill) [inline lua/tabnine/binary.lua:47]: Does `table.sort()` work on these objects? have we checked this?

Comment (aarondill) [reply lua/tabnine/binary.lua:47]: Nevermind, having tested myself, it seems that it does, in fact, work. I wonder if internally it's using the __lt/__gt methods...

**Comment by aarondill** [comment]
Only thing I see is that you seem to have forgotten to remove the _actual_ `third_party/semver` (and the whole `third_party` folder can be removed) from the repo

**Review by aarondill — CHANGES_REQUESTED** [review]
Only thing I see is that you seem to have forgotten to remove the actual third_party/semver (and the whole third_party folder can be removed) from the repo

File: lua/tabnine/binary.lua (hunk 1 above)
Comment (amirbilu) [reply lua/tabnine/binary.lua:47]: Oh yes.. it does :)

**Pull request merged**